docs/workspaces.md

```diff
@@ -102,9 +102,13 @@ coder update <workspace-name>
 
 ## Repairing workspaces
 
-Use the following command to re-enter template input variables in an
-existing workspace. This command is useful when a workspace fails to
-build because its state is out of sync with the template.
+You can re-enter template input variables from a workspace. This is
```

Comment:
I am a bit confused here on linking template wide variables with the workspace. 
@bpmct can you clear this confusion?
Also this paragraph seems a bit out of flow.

Reply:
I agree. I pushed some edits to move this towards the bottom and also mention `coder state <push/pull>`